### DISCUSSION TITLE
rabbitmq image issue on docker swarm (not-privileged)

### DISCUSSION THREAD

**Discussion by box-daxter** [docker-library/rabbitmq#499] [Q&A] [answered]
Hi Mates, 

I'm dealing with this since few days ago, I'm trying to run rabbtimq on Docker-swarm cluster of Rapsberry Pi 4.

The main issue is:
```
root@rp04:~# docker run -it rabbitmq:3.8.17-management-alpine sh
/ # rabbitmq-server 
clock_gettime(CLOCK_MONOTONIC, _) failed: Operation not permitted (1)
Aborted (core dumped)
/ # 
```

Some idea? 3.7.29 is working properly.

**Comment by yosifkit** [comment, marked as answer]
https://github.com/docker-library/php/issues/1172#issuecomment-869912523:
> Looks like you are all running into the exact issue our build servers were hitting last week. If Alpine 3.13 works but Alpine 3.14 fails with an "operation not permitted", then, in this instance, the issue is not the image. You are hitting what https://github.com/opencontainers/runc/pull/2750 fixes, so you need an updated docker; `20.10.4` and above includes the updated `runc`.

Discussion from https://github.com/docker-library/rabbitmq/issues/488#issuecomment-869978072:
> on the raspberry pi i already had docker version 20.10.7 (build f0df350)

> I had the same issue, fixed it by updating libseccomp2 on my pi 4

So you may need a newer Docker (to get a new `runc`) and a newer `libseccomp2`.

related issues: https://github.com/docker-library/ruby/issues/351, https://github.com/docker-library/rabbitmq/issues/488
